src/kOS/Function/PrintList.cs

```diff
@@ -276,7 +273,7 @@ private kList GetConfigList()
             list.AddColumn("Name", 34, ColumnAlignment.Left);
             list.AddColumn("Value", 6, ColumnAlignment.Left);
 
-            foreach (ConfigKey key in Config.Instance.GetConfigKeys())
+            foreach (ConfigKey key in SafeHouse.Config.GetConfigKeys())
```

Comment:
The interface exposed by `SafeHouse.Config` does not include `GetConfigKeys()`.  I'm going to revert this back to `Config.Instance` in the merge.